ApplicationInsights.config

```diff
@@ -1,6 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <ApplicationInsights schemaVersion="2014-05-30" xmlns="http://schemas.microsoft.com/ApplicationInsights/2013/Settings">
-  <InstrumentationKey>4b9bb17b-c7ee-43e5-b220-ec6db2c33373</InstrumentationKey>
   <TelemetryInitializers>
```

Comment:
Who is now putting this file into the bin folder? Seems like none of the projects are copying it?

Reply:
This file is no longer necessary and has been deleted.